liquigraph-core/src/main/java/org/liquigraph/core/io/xml/LiquigraphEntityResolver.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2014-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.core.io.xml;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.EntityResolver;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+/**
+ * Resolves XML entities against local classpath resources.
+ */
+public class LiquigraphEntityResolver implements EntityResolver {
```

Comment:
minor: rename to `LiquigraphLocalEntityResolver`?

Reply:
note that it could be tested with sth like https://sniffy.io/docs/latest/#_simulating_no_connectivity_in_unit_tests. Worth a try.

Reply:
Will get to renaming.
Sniffy looks interesting, hadn't come across that before.  Will see if I can get some time to try adding a test with it.

Reply:
The latest change has the renaming in - haven't dug into the testing yet.